appsync-eventbridge-datasource/README.md

```diff
@@ -0,0 +1,55 @@
+# Appsync to EventBridge Datasource 
+
+This pattern demonstrates sending events directly to Event Bus by integrating Appsync with Eventbridge Datasource.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/appsync-eventbridge-datasource
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+- [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+- [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+   ```bash
+   git clone https://github.com/aws-samples/serverless-patterns
+   ```
+2. Change directory to the pattern directory:
+   ```bash
+   cd serverless-patterns/appsync-eventbridge-datasource/cdk
+   ```
+3. Install dependencies:
+   ```bash
+   npm install
+   ```
+4. From the command line, configure AWS CDK:
+   ```bash
+   cdk bootstrap ACCOUNT-NUMBER/REGION # e.g.
+   cdk bootstrap 1111111111/us-east-1
+   cdk bootstrap --profile test 1111111111/us-east-1
+   ```
+5. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the `lib/cdk-stack.ts` file:
+   ```bash
+   npm run build && cdk deploy
+   ```
+
+## How it works
+
+Let's submit mutation request through AWS Appsync Console. Appsync valiate the schema and publish the message directly to Event Bus using the Appsync JS Resolver.
```

Comment:
be more specific with the tests
all tests need to be done as CLI if possible, so please add command line examples or screenshots and step by step on how to test and how to see the results.

Reply:
Updated the Readme file